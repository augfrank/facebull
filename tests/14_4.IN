M0   C0   C4  2562
M1   C0   C5  2345
M2   C0   C6  2675
M3   C0   C11  2570
M4   C0   C13  2244
M5   C1   C3  2528
M6   C1   C6  2688
M7   C1   C7  2880
M8   C1   C8  2967
M9   C1   C10  2774
M10   C1   C11  2098
M11   C1   C12  2783
M12   C2   C0  2267
M13   C2   C3  2368
M14   C2   C5  2699
M15   C2   C6  2554
M16   C2   C11  2442
M17   C2   C12  2375
M18   C2   C13  2397
M19   C3   C0  2216
M20   C3   C5  2958
M21   C3   C8  2213
M22   C3   C10  2705
M23   C3   C11  2052
M24   C4   C0  2515
M25   C4   C1  2026
M26   C4   C2  2678
M27   C4   C3  2310
M28   C4   C6  2481
M29   C4   C7  2927
M30   C4   C9  2635
M31   C4   C10  2248
M32   C4   C13  2476
M33   C5   C1  2524
M34   C5   C2  2630
M35   C5   C4  2135
M36   C5   C7  2000
M37   C5   C8  2786
M38   C5   C10  2944
M39   C5   C12  2089
M40   C6   C0  2365
M41   C6   C1  2996
M42   C6   C3  2885
M43   C6   C4  2414
M44   C6   C5  2366
M45   C6   C9  2293
M46   C6   C11  2024
M47   C7   C1  2467
M48   C7   C4  2272
M49   C7   C5  2102
M50   C7   C12  2056
M51   C7   C13  2958
M52   C8   C1  2310
M53   C8   C5  2740
M54   C8   C6  2546
M55   C8   C9  2142
M56   C8   C10  2386
M57   C8   C12  2863
M58   C8   C13  2456
M59   C9   C0  2446
M60   C9   C4  2653
M61   C9   C6  2028
M62   C9   C8  2586
M63   C9   C10  2416
M64   C9   C11  2832
M65   C9   C12  2361
M66   C10   C0  2551
M67   C10   C3  2839
M68   C10   C4  2360
M69   C10   C6  2829
M70   C10   C7  2684
M71   C10   C8  2624
M72   C10   C13  2531
M73   C11   C2  2495
M74   C11   C3  2655
M75   C11   C5  2002
M76   C11   C9  2666
M77   C11   C13  2874
M78   C12   C1  2226
M79   C12   C2  2073
M80   C12   C4  2821
M81   C12   C9  2863
M82   C12   C10  2900
M83   C12   C13  2050
M84   C13   C5  2020
M85   C13   C6  2868
M86   C13   C8  2852
M87   C13   C11  2170
