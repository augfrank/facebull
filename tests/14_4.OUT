30903
4 10 13 21 25 36 40 50 56 61 68 76 79 84
